packages/vue/src/components/atoms/SfBadge/SfBadge.vue

```diff
@@ -1,5 +1,10 @@
-<template>
-  <div class="sf-badge">
+<template functional>
+  <div
+    :class="[data.class, data.staticClass, 'sf-badge']"
+    :style="[data.style, data.staticStyle]"
+    v-bind="data.attrs"
+    v-on="listeners"
```

Comment:
$listeners or listeners?

Reply:
In this case it should be listeners without $ :) https://vuejs.org/v2/guide/render-function.html#Passing-Attributes-and-Events-to-Child-Elements-Components

Reply:
Should I leave it like it is now?